### PR TITLE
feat: Add attach images UI Tests - Meeds-io/MIPs#53

### PR DESCRIPTION
**Pull request by SaraBoutej**
Included Tests:

 - When sending a kudos, then I can attach images
 - When kudos posted, then images attached to it are displayed below the message
 - When adding an action, I can announce its availability and attach images to the post
 - When achieving an action, I can attach images to the message
 - When posting comment with images, then images are listed below the comment
 - When posting a message with images, then slider proposed only when more than one image
 - When I have added images to a message AND I edit the message, then I can remove or edit the images
 - When I edit a message, then I can add an image
 - When adding a gif and accessing the edit drawer, I can't edit the GIF
 - When I edit a message, then I can add an image
 - When images added to the description and this edition is applied, then images are listed below the description
 - When editing the description, if no changes to images, then when applying, no changes applied
 - When editing the description, if I attach images to it, then when applying, images attached to it
 - When editing the description and clicking outside without applying changes, then images are not saved
 - When adding images to a task comment, then it is listed below the task comment as expected

### CODE REVIEW

**Review by boubaker — COMMENTED** [review]
Automatic Tests fails.

File: src/test/resources/features/Social/AttachImage.feature [hunk 1]
```diff
@@ -16,31 +16,307 @@ Feature: Attach images activities
...
+    And The attached image is displayed in the task description
+    
+    When I click to edit task description
+    And I enter description for task 'Task Description attached images edited'
+    And I attach a second image to the task description
+    
+    When I click outside the task description
+    Then The new attached image is not displayed
+    
+  @test
```
Comment (boubaker) [inline src/test/resources/features/Social/AttachImage.feature]:
```suggestion
```

**Review by boubaker — COMMENTED** [review]
@SaraBoutej please execute this branch in CI to ensure that this will not make the CI/CD fails once merged with the associated PRs

File: src/test/resources/features/Social/AttachImage.feature (hunk 1 above)
Comment (SaraBoutej) [reply src/test/resources/features/Social/AttachImage.feature]: removed

**Comment by SaraBoutej** [comment]
Build success for new Test cases 
https://ci.exoplatform.org/view/PLF%20-%20QA%20UI/job/exo-qa-ui/job/qa-ui-auto-meeds/649/

**Review by boubaker — APPROVED** [review]

**Pull request merged**